docs/kb/band.md

```diff
@@ -3,105 +3,185 @@ order: 12
+
+In the `x/oracle/types/keys.go` file, update the `Version` variable in the `const` block to the required version that the IBC module supports:
 
 ```go
-const Version = "bandchain-1"
+const (
+    
+    // Version defines the current version the IBC module supports
+    Version = "bandchain=1"
 ```
 
-After scaffold and change the data, run the chain:
+## Start your chain in development
+
+To run the chain from the `oracle` directory:
+
 ```shell
-$ starport chain serve
+starport chain serve
 ```
 
-In another tab, configure and run the starport relayer.
-```shell
-$ starport relayer configure -a \
+Keep this terminal window open.
+
+## Configure and connect the Starport relayer
+
+In another terminal tab, configure the [Starport relayer](../docs/kb/../../kb/relayer.md):
+
+```bash
+starport relayer configure -a \
 --source-rpc "http://rpc-laozi-testnet4.bandchain.org:80" \
 --source-faucet "https://laozi-testnet4.bandchain.org/faucet" \
 --source-port "oracle" \
 --source-gasprice "0uband" \
 --source-gaslimit 5000000 \
 --source-prefix "band" \
---source-version "bandchain-1" \
+--source-version "bandchain=1" \
 --target-rpc "http://localhost:26657" \
 --target-faucet "http://localhost:4500" \
 --target-port "consuming" \
 --target-gasprice "0.0stake" \
 --target-gaslimit 300000 \
 --target-prefix "cosmos"  \
---target-version "bandchain-1"
-            
-$ starport relayer connect
+--target-version "bandchain=1"
+```
+
+When prompted, press Enter to accept the default source and target accounts.
+
+The command output confirms the relayer is successfully configured:
+
+```bash
+? Source Account default
+? Target Account default
+
+🔐  Account on "source" is default(band1dscvlx0mhpys9fazuk7ej9z4cq7qknzn09pjpq)
+
+ |· received coins from a faucet
+ |· (balance: 10000000uband)
+
+🔐  Account on "target" is default(cosmos1dscvlx0mhpys9fazuk7ej9z4cq7qknznk2pseg)
+
+ |· received coins from a faucet
+ |· (balance: 100000stake,5token)
+
+⛓  Configured chains: band-laozi-testnet4-oracle
 ```
 
-Open one more terminal tab to make a request transaction, passing the script id.
+Connect the relayer:
+
+```bash
+starport relayer connect
+```
+
+Leave this terminal tab open so you can monitor the relayer.
+
+## Make a request transaction
+
+In another terminal tab, use the `oracled` binary to make a request transaction. Because BandChain has multiple scripts already deployed into the network, you can request any data using the BandChain script id. In this case, use script 37 for Coin Rates:
+
 ```shell
 # Coin Rates (script 37 into the testnet)
-$ oracled tx consuming coin-rates-data 37 4 3 --channel channel-0 --symbols "BTC,ETH,XRP,BCH" --multiplier 1000000 --fee-limit 30uband --prepare-gas 600000 --execute-gas 600000 --from alice --chain-id oracle
+oracled tx consuming coin-rates-data 37 4 3 --channel channel-0 --symbols "BTC,ETH,XRP,BCH" --multiplier 1000000 --fee-limit 30uband --prepare-gas 600000 --execute-gas 600000 --from alice --chain-id oracle
```

Comment:
@Pantani  I must be missing something because this command didn't work for me:

What is "coin-rates-data"? 

Error: unknown command "coin-rates-data" for "consuming"

Reply:
@Pantani I know you tested and did not get errors, so let's merge this PR